### DISCUSSION TITLE
Wi-Fi without authentication?

### DISCUSSION THREAD

**Discussion by winnieXY** [richonguzman/LoRa_APRS_iGate#179] [General]
Hey! 

I'll be abroad next week and I know that there is no iGate next to me, but a open Wi-Fi. Is it possible to use the tracker with a Wi-Fi without authentication by specifing just the WiFi name an no password? 

Thanks in advance!

**Comment by richonguzman** [comment]
Tracker is ment to do all of its connections over LoRa.

the iGate can connect to a public Wifi just omitting the password

maybe if no igate is available around a portable igate is needed or just use any phone with a aprs app ?

**Reply by winnieXY** [reply]
Sorry, I meant the iGate. I have a place with open Wi-Fi where I would like to temporarily place the iGate.

**Reply by richonguzman** [reply]
igate firmware should work as with any WIFI

the only thing I dont know is :
 "would the firmware connect to a wifi that has no password request with just not writing the password on the web config page of the igate"?"???

**Reply by dl1lmh** [reply]
Well, i think, the bigger problem of public WiFi might be the "landing page" to accept Terms of use...

**Reply by ghost** [reply]
> igate firmware should work as with any WIFI
> 
> the only thing I dont know is : "would the firmware connect to a wifi that has no password request with just not writing the password on the web config page of the igate"?"???

I became interested in this topic after using an iGate and a tracker from you @richonguzman for quite some time. Thank you for your work.

Connecting to a Wi-Fi network that does not have a password stored for the SSID works perfectly. I played around with different SSIDs and the connection was always established.

**Reply by ghost** [reply]
> Well, i think, the bigger problem of public WiFi might be the "landing page" to accept Terms of use...

Yes, that's correct. Unfortunately, that is not possible (yet). I have tested that as well. Since the connection data is stored and then processed, no credential request appears in the WebGUI.
